.idea/workspace.xml

```diff
@@ -2,7 +2,12 @@
 <project version="4">
```

Comment:
The project really needs a .gitignore file so we don't check in all these extra files - give me a shout if you need a hand figuring it out.